package.json

```diff
@@ -10,9 +10,6 @@
     "@types/node": "^16.7.13",
     "@types/react": "^18.0.0",
     "@types/react-dom": "^18.0.0",
```

Comment:
My bad: all types (including TypeScript) should be stored only in `devDependencies` since they are not taking any role in final build. Let's move it to corresponding section, please